### PR TITLE
Add textarea var type

### PR DESCRIPTION
**Pull request by marc-gr**
## What does this PR do?

Adds a new textarea var type.

## Why is it important?

For QoL and better UX

## Checklist

~- [ ] I have added test packages to [`test/packages`](https://github.com/elastic/package-spec/tree/main/test/packages) that prove my change is effective.~
- [x] I have added an entry in [`versions/N/changelog.yml`](https://github.com/elastic/package-spec/blob/main/versions/1/changelog.yml).

## Related issues

- Relates https://github.com/elastic/kibana/pull/133070

### CODE REVIEW

File: versions/1/changelog.yml [hunk 1]
```diff
@@ -7,6 +7,9 @@
     - description: Prepare for next patch release
       type: enhancement
       link: https://github.com/elastic/package-spec/pull/353
+    - description: Add 'textarea' var type
+      type: enhancement
+      link: https://github.com/elastic/package-spec/pull/
```
Comment (mtojek) [inline versions/1/changelog.yml:12]: nit: PR number :)

**Comment by mtojek** [comment]
Assigning Josh as this is mostly related to Fleet.

**Comment by joshdover** [comment]
Replacing my review with @kpollich, though I do wonder if we can assign these reviews to the general @elastic/fleet team to have less of a bottleneck on tech leads.

**Review by kpollich — APPROVED** [review]
I left some comments over on https://github.com/elastic/kibana/pull/133070, but in terms of the actual spec changes I have no real qualms here. Thanks for seeing the work through both in Kibana and over here in the spec 👍

**Review by mtojek — DISMISSED** [review]

**Review by mtojek — APPROVED** [review]
LGTM!

**Pull request merged**